### PR TITLE
[Docs] Grid Documentation

### PR DESCRIPTION
**Pull request by jamesb93**
Updates the text of Grid which was, IMO, too wordy and compacted too much info into the description. I took the liberty of moving much of the more detailed stuff to the discussion instead.﻿

### CODE REVIEW

**Review by weefuzzy — APPROVED** [review]
Perhaps 'obtain' could be 'generate', and the gnarly detail of Jonker-Volegenant could shuffle into the discussion as well. However, nothing to substantively stop this merging.

**Comment by jamesb93** [comment]
> Perhaps 'obtain' could be 'generate', and the gnarly detail of Jonker-Volegenant could shuffle into the discussion as well. However, nothing to substantively stop this merging.

Cool I've made those changes. Thanks!

**Review by tedmoore — APPROVED** [review]

**Review by tremblap — APPROVED** [review]
Indeed much, much cleaner!

**Comment by jamesb93** [comment]
Just on another note to myself, there quite a few docs that could benefit from seeing if discussion/description can be separated out more. I noticed some have no description so there is wiggle room there to rejig some of the info.
